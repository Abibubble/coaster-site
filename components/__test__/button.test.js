import React from 'react'
import renderer from 'react-test-renderer'
import Button from '../button'

describe('Button', () => {
  it('should render button component', () => {
    const tree = renderer.create(<Button>Button</Button>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
