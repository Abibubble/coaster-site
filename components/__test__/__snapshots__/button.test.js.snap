// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button component 1`] = `
<button
  className="main"
  rel=""
  target="_self"
>
  <span>
    Button
  </span>
</button>
`;
